packages/xo-web/src/xo-app/backup-ng/index.js

```diff
@@ -53,54 +55,80 @@ const _runBackupNgJob = ({ id, name, schedule }) =>
+          ) {
+            lastRunLog = log
+          }
+        }
+        cb(lastRunLog)
+      }),
+  })),
+  ({ job, schedule, lastRunLog }) => (
+    <Ul>
+      {schedule.name ? (
+        <Li>{_.keyValue(_('scheduleName'), schedule.name)}</Li>
+      ) : (
+        <Li>{_.keyValue(_('scheduleCron'), schedule.cron)}</Li>
+      )}
+      <Li>
+        <StateButton
+          disabledLabel={_('stateDisabled')}
+          disabledHandler={enableSchedule}
+          disabledTooltip={_('logIndicationToEnable')}
+          enabledLabel={_('stateEnabled')}
+          enabledHandler={disableSchedule}
+          enabledTooltip={_('logIndicationToDisable')}
+          handlerParam={schedule.id}
+          state={schedule.enabled}
+          style={{ marginRight: '0.5em' }}
+        />
+        {job.runId !== undefined ? (
+          <ActionButton
+            btnStyle='danger'
+            handler={cancelJob}
+            handlerParam={job}
+            icon='cancel'
+            key='cancel'
+            size='small'
+            tooltip={_('formCancel')}
+          />
         ) : (
-          <Li>{_.keyValue(_('scheduleCron'), schedule.cron)}</Li>
-        )}
-        <Li>
-          <StateButton
-            disabledLabel={_('stateDisabled')}
-            disabledHandler={enableSchedule}
-            disabledTooltip={_('logIndicationToEnable')}
-            enabledLabel={_('stateEnabled')}
-            enabledHandler={disableSchedule}
-            enabledTooltip={_('logIndicationToDisable')}
-            handlerParam={schedule.id}
-            state={schedule.enabled}
-            style={{ marginRight: '0.5em' }}
+          <ActionButton
+            btnStyle='primary'
+            data-id={job.id}
+            data-name={job.name}
+            data-schedule={schedule.id}
+            handler={_runBackupNgJob}
+            icon='run-schedule'
+            key='run'
+            size='small'
           />
-          {job.runId !== undefined ? (
-            <ActionButton
-              btnStyle='danger'
-              handler={cancelJob}
-              handlerParam={job}
-              icon='cancel'
-              key='cancel'
-              size='small'
-              tooltip={_('formCancel')}
-            />
-          ) : (
-            <ActionButton
-              btnStyle='primary'
-              data-id={job.id}
-              data-name={job.name}
-              data-schedule={schedule.id}
-              handler={_runBackupNgJob}
-              icon='run-schedule'
-              key='run'
-              size='small'
-            />
-          )}
-        </Li>
-      </Ul>
-    ))}
-  </div>
-)
+        )}{' '}
+        {lastRunLog !== undefined && (
+          <ActionButton
+            handler={showTasks}
+            handlerParam={lastRunLog}
+            icon='preview'
+            size='small'
+            tooltip={_('scheduleLastRun')}
+            className={`text-${STATUS_LABELS[lastRunLog.status].className}`}
```

Comment:
Would `btn-` work instead? (So it looks more like the status pills in the log table)